docs/Report.md

```diff
@@ -151,12 +151,21 @@ Before putting anything into the workflow actions, we create test manually to ru
 
-## Team work
-Show a screenshot of your project board right before hand-in. Briefly describe which tasks are still unresolved, i.e., which features are missing from your applications or which functionality is incomplete.
+# Teamwork
+This section will describe what features and implementation weren't completed and how the group worked with creation of issues and development. <br>
 
-Briefly describe and illustrate the flow of activities that happen from the new creation of an issue (task description), over development, etc. until a feature is finally merged into the main branch of your repository.
+## Project Board
+![Fix.XX Project Board](Images/ProjectBoard.png){width=60%}
+<br>
+This figure shows the Project board of Chirp at the day of hand-in. We have four issues that haven't been implemented before the deadline. All the four issues are under the Todo section but none of them are in the project requirement. That is to say they were formulated under the *Wild Style* development section. They were, adding tags to cheeps, being able to cheep a meme, trending cheeps and a re-cheep feature. As can be seen from the project board the re-cheep feature had an assigned developer but wasn't completed in time as other requirements had to be met. One Wild style feature was implemented, a like button for the cheeps. Although the like implementation is missing some functionality. A user can't see which cheep they've like and they can like a cheep infinitely many times.
+
+## Issue creation
+![Fig.XX Flow of issues](Images/teamwork.png){width=60%}
+<br>
+This illustration show the flow of our work-process when being met with new requirements for the project from the lectures. At first the new requirements are read and understood, then the group gathers and tries to formulate the tasks into small issues which ideally can be completed within a days worth of work. If a formulation gets accepted by the group it gets posted on the issue board on github. A developer assigns themselves to an issue to let the others know what they are working on. When the developer feels like they've implemented the feature adequately they commit and create a pull request. When a pull request is posted 2 reviewers from the group are needed to further merge it to main and deploy. When reviewing the code a reviewer can request changes and then further work on the issue is required. This process repeats until two reviewers accept the changes and then the code can be merged with main. <br>
+
+Another more simple *"issue-creation-flow"* was also used. If a developer found a bug with the application or other small adjustment were made an issue was created. This created good documentation for the other developers so all group members could understand why a pull request was made. 
```

Comment:
Another more simple *"issue-creation-flow"* was also used. If a developer found a bug within the application or other small adjustments were made. An issue was created. This created good documentation for the other developers so all group members could understand why a pull request was made.

Reply:
Thanks